src/main/scala/uk/gov/hmrc/ct/computations/offPayRollWorking/package.scala

```diff
@@ -9,8 +9,7 @@ import org.joda.time.LocalDate
 import uk.gov.hmrc.ct.box.EndDate
 
 package object offPayRollWorking {
-
-  val opwApplies2020 = LocalDate.parse("2020-04-06")
+  val opwApplies2020 = new LocalDate(2017, 4, 6)
```

Comment:
Cant we just use LocalDate.parse like before?

Reply:
Yes